drivers/SmartThings/zigbee-button/src/zigbee-multi-button/init.lua

```diff
@@ -57,7 +59,7 @@ local function added_handler(self, device)
     end
     device:emit_component_event(component, capabilities.button.numberOfButtons({value = number_of_buttons}, {visibility = { displayed = false }}))
   end
-  -- device:emit_event(capabilities.button.button.pushed({state_change = false}))
+  device:emit_event(capabilities.button.button.pushed({state_change = false}))
```

Comment:
This was intentionally commented out for the migration period and needs to stay that way for now, sorry.